packages/select-rich/src/LionSelectRich.js

```diff
@@ -200,31 +197,17 @@ export class LionSelectRich extends OverlayMixin(
     this.__setupInvokerNode();
     this.__setupListboxNode();
 
-    this._invokerNode.selectedElement = this.formElements[this.checkedIndex];
+    // Now that we have rendered our listbox, try setting the user defined modelValue again
```

Comment:
Bit hacky, I tried using updateComplete but then you combine setting child.checked with updateComplete which causes infinite loops. This is because a model-value-changed event is fired due to toggling child.checked and that bubbles back to your rich select causing another update, etc. etc. until infinity.. So just caching the not found modelValue and syncing down after firstUpdated is pretty clean in comparison 🙈